Trying to obtain a footer on a grid that has no footer gives NPE

Given that footers are optional it would be great to return null instead of throwing the following NPE:

```
Caused by: java.lang.NullPointerException
    at jdk.internal.reflect.UnsafeFieldAccessorImpl.ensureObj (UnsafeFieldAccessorImpl.java:57)
    at jdk.internal.reflect.UnsafeObjectFieldAccessorImpl.get (UnsafeObjectFieldAccessorImpl.java:36)
    at java.lang.reflect.Field.get (Field.java:418)
    at com.flowingcode.vaadin.addons.gridhelpers.HeaderFooterGridHelper.getTemplate (HeaderFooterGridHelper.java:22)
    at com.flowingcode.vaadin.addons.gridhelpers.HeaderFooterGridHelper.getFooter (HeaderFooterGridHelper.java:64)
    at com.flowingcode.vaadin.addons.gridhelpers.GridHelper.getFooter (GridHelper.java:308)
```

--- a/src/main/java/com/flowingcode/vaadin/addons/gridhelpers/HeaderFooterGridHelper.java
+++ b/src/main/java/com/flowingcode/vaadin/addons/gridhelpers/HeaderFooterGridHelper.java
@@ -61,7 +61,7 @@ class HeaderFooterGridHelper implements Serializable {
 
   public String getFooter(@NotNull Grid.Column<?> column) {
       final Renderer<?> headerRenderer = getFooterRenderer(column);
-      return getTemplate(headerRenderer);
+      return headerRenderer==null?null:getTemplate(headerRenderer);
   }
 
 }

